package.json

```diff
@@ -52,17 +52,17 @@
     "nyc": "15.1.0"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
   },
   "dependencies": {
     "ajv": "8.1.0",
     "clone": "2.1.2",
     "commander": "8.2.0",
     "fast-xml-parser": "3.21.1",
-    "libxmljs2": "0.32.0",
+    "libxmljs2": "0.35.0",
```

Comment:
@parthverma1 It seems `0.33.0` drops support for node v19 ([ref](https://github.com/marudor/libxmljs2/releases/tag/v0.33.0)).

We do have unit tests ([ref](https://github.com/postmanlabs/wsdl-to-postman/blob/69dc1be8226ce7f5d851730246fae276321c3c1d/test/unit/LibXMLjs2Facade.test.js#L32)) that make sure basic functionality we consume works correctly so it should be fine with v18 I think but I'll test it out manually once as well and provide update here.

Also I couldn't find any detailed changelogs in libxmljs2 repo that could give us idea of any changes that could affect us but most of them are mostly to keep it in maintenance mode and support new version or drop old version support.